podcast/2.18.md

```diff
@@ -0,0 +1,545 @@
+
+And so I responded to the ... I basically quickly wrote a little program that just hit the API with all hundred flags, dumped the results. And I happened to capture the timing for how long it took to do the search. So I printed out the results in a nice format. I put it in the email. I responded and I said, "By the way, I know there'll be questions about what produced these results. I've been not really telling people yet that I've been working on this little tool, and I'll tell y'all more about it if you want to, but here's the two-sentence description of it." And people were quite excited to have the immediate answer in the email thread, but I also had people reach out to me very interested in working on the tool.
+
+One of them was Jonathan Klein, who's the other person who worked on Hound with me, who was like, "I'm just going to build a UI. Is that okay?" I'm like, "Yes, please go build a UI." So Jonathan would end up building the first UI for Hound. And yeah, it got really popular in Etsy. And in fact, I still have folks from Etsy who reach out to me and tell me they still use Hound daily. MailChimp still uses Hound daily. And people in Mailchimp often thank me for it, which is really interesting because I feel like the tool is really more of hitting a sweet spot between what existed. Like it was just a quick path to take something that existed and to put a web UI on it that really captured the things that a web UI would give you: the ability to update on every commit to the repo and to link to search results. The linking to search results turned out to be way bigger than anyone expected. Like most of Hound's traffic is clicking on links.
+
+**Beyang Liu:**:
+
+Yeah. Your experience resonates just so much from the kind of initial skepticism. Like we hear all the time. I almost divide the world into two camps of developers: those who get code search, and those who've never tried it before. And the ones who haven't tried it really just have a hard time comprehending how this is better than grep, or better than control-F in the editor. 
+
+And it is one of those things where, once you start using it, you start using it more and more. It's awesome to have all the code searchable in one place. And then the link sharing too, it just becomes almost this ... Like it starts out as a single-player tool because you're just searching for stuff for yourself. But then you start sharing links with colleagues. They can jump to what you're looking into and continue the exploration from there. It's kind of interesting that it's not more, I guess, universal at this point. I would say like Etsy and MailChimp are kind of the exception. Most engineering organizations don't have a code search tool, which is which interesting, no?
```

Comment:
```suggestion
And it is one of those things where, once you start using it, you start using it more and more. It's awesome to have all the code searchable in one place. And then the link sharing too, it just becomes almost this ... Like it starts out as a single-player tool because you're just searching for stuff for yourself. But then you start sharing links with colleagues. They can jump to what you're looking into and continue the exploration from there. It's kind of interesting that it's not more, I guess, universal at this point. I would say like Etsy and MailChimp are kind of the exception. Most engineering organizations don't have a code search tool, which is interesting, no?
```